### PR TITLE
Added all fixes proposed by Gayle.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: ng-installation.adoc [hunk 1]
```diff
@@ -15,90 +15,114 @@ still available via the usual CLI commands.
...
+ +
+Furthermore, the switch might lead to an abrupt increase in the number of connections to the server, and consequently its load, due to the resynchronisation of all devices.*
 
 The switch is completely transparent to end users, and no user interaction
- will be prompted or required.
+ should be prompted or required, but being the Exchange ActiveSync protocol mostly client-driven different behaviours might be experienced, such as:
+
+* Device not synchronising until user's action (e.g. opening the email client).
+* Device requiring a restart.
+* Device not synchronising until the user re-enter its username and password in the account's settings.
```
Comment (fatkudu) [inline ng-installation.adoc]: Suggest change "user re-enter its" to "user re-enters their"

**Review by fatkudu — CHANGES_REQUESTED** [review]

**Review by gaylebillat — APPROVED** [review]
Looks good to me! Thanks!

**Comment by gordyt** [comment]
Published a preview export of this pull request.  It affected two documents, single-server and multi-server install guides.  Previews may be viewed here:

* [Single Server Install Guide](https://zimbra.github.io/installguides/work-in-progress/single.html)
* [Multi Server Install Guide](https://zimbra.github.io/installguides/work-in-progress/multi.html)

*Note:* I created a `bugfix/zextras` branch based off of `develop` (not `master`, as all pull request from contributors should be issued against the `develop` branch) that contained the commits from the pull request.

Please review for content.

File: ng-installation.adoc (hunk 1 above)
Comment (michaelmedellin) [reply ng-installation.adoc]: @zextras-contrib have these changes suggested by @fatkudu been merged into this PR?

**Review by michaelmedellin — APPROVED** [review]

**Comment by gordyt** [comment]
Pull request has been merge. Documentation exports have been updated and may be seen [here](https://zimbra.github.io/installguides/).  Use either the _latest version_ or the _8.8.5_ links.

**Pull request closed without merging**